datasets/lv/sanction_list/lv_sanction_list.yml

```diff
@@ -0,0 +1,28 @@
+name: lv_sanction_list
+title: Latvia List of Sanctions
+entry_point: crawler.py
+prefix: lv-sanction
+summary: List of Sanctions in Latvia
+description:
+  This dataset contains lists of individuals or entities subject to international or national sanctions as specified by laws in Latvia
+
+  International sanctions are imposed on subject in accordance to international laws as set by organizations such as the UN, EU or other international bodies.
+
+  National sanctions are determined by Latvia's legislative acts and international laws. They are imposed by the Cabinet of Ministers following procedures outlined
+  in the Law on International Sanctions and National Sanctions of the Republic of Latvia.
+
+  It's important to note that the dataset only includes information on financial restrictions imposed on these individuals or entities and does not cover other types
+  of sanctions like restrictions on the circulation of goods.
+
+url: https://sankcijas.fid.gov.lv
+publisher:
+  name: Finanšu izlūkošanas dienests
+  description:
+    The Financial Intelligence Unit maintains information on the subjects of sanctions. The information is gotten from the [United Nations Security Council](https://www.un.org/securitycouncil/content/un-sc-consolidated-list)
+    and the [European Union(https://webgate.ec.europa.eu).
+    The Financial Intelligence Unit provide actual information without making any changes to the original source and is not responsible for any gaps or errors that occurs from the creators.
```

Comment:
I think you can use this description straight from https://fid.gov.lv/en instead - that describes the publisher better for me. You can prefix the lines with > to indicate it's a quote. And then use https://fid.gov.lv/en as the publisher link

> The Financial Intelligence Unit of Latvia is an independent authority and it's responsibility lies with prevention of money laundering in Latvia, with the goal of eliminating the possibility of using the Latvian financial system to launder money or finance terrorism.